packages/api-clients/template.hbs

```diff
@@ -77,7 +78,16 @@ export const {{@key}}Endpoints = makeApi([
 export const {{@key}}Api = new Zodios({{#if options.baseUrl}}"{{options.baseUrl}}", {{/if}}{{@key}}Endpoints);
 
 export function create{{pascalcase @key}}ApiClient(baseUrl: string, options?: ZodiosOptions) {
-    return new Zodios(baseUrl, {{@key}}Endpoints, options);
+    return new Zodios(baseUrl, {{@key}}Endpoints, {
+			...options,
+			axiosConfig: {
+				...options?.axiosConfig,
+				// This configuration is used to serialize properly query parameter name when value is array  
+				// the default client serialization produce query param name appending the invalid chars "[]"	at the end
+				// eg: ids[]=1,2,3 instead of ids=1,2,3
```

Comment:
```suggestion
				// This configuration is used to serialize correctly array query parameters.
				// The default serialization produces a query param name appending "[]" after the equals:
				// eg: ids[]=1,2,3 instead of ids=1,2,3
```